app/pages/sections/transport/TransportMeansActiveSection.scala

```diff
@@ -16,10 +16,11 @@
 
 package pages.sections.transport
 
+import models.Index
 import pages.sections.Section
 import play.api.libs.json.{JsObject, JsPath}
 
-case object TransportMeansActiveSection extends Section[JsObject] {
+case class TransportMeansActiveSection(index: Index) extends Section[JsObject] {
```

Comment:
The path in this case class will need to pass the index into some array. See [CountriesOfRoutingSection](https://github.com/hmrc/manage-transit-movements-departure-frontend/blob/main/app/pages/sections/routeDetails/routing/CountriesOfRoutingSection.scala) and [CountryOfROutingSection](https://github.com/hmrc/manage-transit-movements-departure-frontend/blob/main/app/pages/sections/routeDetails/routing/CountryOfRoutingSection.scala) for an example